docs/03-tutorials/00-serverless/svls-11-set-asynchronous-connection-of-functions.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Set asynchronous communication between Functions
+---
+
+This tutorial demonstrates how to connect two Functions asynchronously. It is based on the [in-cluster Eventing example](https://github.com/kyma-project/examples/tree/main/incluster_eventing).
+
+The example provides a very simple scenario of asynchronous communication between two Functions. The first Function accepts the incoming traffic via HTTP, sanitizes the payload, and publishes the content as an in-cluster event using [Kyma Eventing](https://kyma-project.io/docs/kyma/latest/01-overview/main-areas/eventing/).
+The second Function is a message receiver. It subscribes to the given event type and stores the payload.
+
+It shows only one possible use case. You can find plenty of use cases yourself on how to orchestrate your application logic into specialised Functions and benefit from decoupled, re-usable components and event-driven architecture.
```

Comment:
```suggestion
This tutorial shows only one possible use case. There are many more use cases on how to orchestrate your application logic into specialized Functions and benefit from decoupled, re-usable components and event-driven architecture.
```

Reply:
I rephrased it a bit, because ` You can find plenty of use cases yourself...` brings up a question `where can I find it?` and I assume we just want to point out that there are other use cases, not provide the users with them.